src/app/pages/ArticlePage/Byline/index.test.tsx

```diff
@@ -1,35 +1,39 @@
 /* eslint-disable no-template-curly-in-string */
 import React, { PropsWithChildren } from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, act } from '@testing-library/react';
 import Byline from '.';
 import { ServiceContextProvider } from '../../../contexts/ServiceContext';
 import ArticleTimestamp from '../../../legacy/containers/ArticleTimestamp';
 import {
   bylineWithNoRole,
   bylineWithNoAuthor,
   bylineWithNameAndRole,
+  bylineWithLink,
 } from './fixture';
+import ThemeProvider from '../../../components/ThemeProvider';
 
 interface Props {
+  service: any;
```

Comment:
Similar to above, you can use the `Services` type from the global type definitions

Reply:
Done